assignment2/src/com/assignment2/User.java

```diff
@@ -0,0 +1,86 @@
+package com.assignment2;
+import java.io.*;
+import java.util.*;  
+public class User implements Comparable<User>,Serializable{
+	private static final long serialVersionUID = -7365823861318693364L;
+	private String fullName;
+	private int age;
+	private String address;
+	private int rollNo;
+	ArrayList<String> courses=new ArrayList<String>();
+	static int choice1;
```

Comment:
Generic name given to variables hinders code readability. Better to use names that can describe the options.